scripts/db.dropcolumn/db.dropcolumn.py

```diff
@@ -120,14 +120,14 @@ def main():
             coltypes = ", ".join(coltypes)
 
             cmds = [
-                "BEGIN TRANSACTION",
+                "BEGIN;",
```

Comment:
The added `;` will result in duplicated `;` since in line 132 the semicolons are added for all elements in the list. Probably just cosmetic, but might be better to not add the `;` here. 

This also applies to line 130, and also `v.db.dropcolumn.py`

Reply:
Thanks!